DBPE2ETests/DBPEndToEndTests.swift

```diff
@@ -37,7 +37,7 @@ final class DBPEndToEndTests: XCTestCase {
     let testUserDefault = UserDefaults(suiteName: #function)!
 
     override func setUpWithError() throws {
-        continueAfterFailure = false
+        continueAfterFailure = true
```

Comment:
Temporary until I get them working again now that staging should be working again